sirepo/auth_db/__init__.py

```diff
@@ -160,6 +162,10 @@ def commit(self):
         self._commit_or_rollback(commit=True)
 
     def create_or_upgrade(self):
+        global _created
+        if _created:
```

Comment:
To prevent being called multiple times in srunit. I think it is a reasonable case since there's significant work that gets done in do_all, and this would slow down all unit tests.